src/openforms/forms/tests/variables/test_viewset.py

```diff
@@ -1018,3 +1018,57 @@ def test_bulk_create_and_update_with_prefill_constraints(self):
                     "Prefill plugin, attribute and options can not be specified at the same time."
                 ),
             )
+
+    def test_bulk_create_and_update_with_non_camel_case_initial_values(self):
+        user = StaffUserFactory.create(user_permissions=["change_form"])
+        self.client.force_authenticate(user)
+
+        form = FormFactory.create()
+        form_step = FormStepFactory.create(form=form)
```

Comment:
FYI we have a trait for the minimal setup in the `Form` factory, so you could do something like this `form = FormFactory.create(generate_minimal_setup=True)`, which will create a form step as well . 

Of course no need to change something here, just to be aware of.

Reply:
Ah okay cool!

To keep things clean, i've changed this to use the `generate_minimal_setup`